### PR TITLE
[UG] Fixed add-c filter-c

### PR DESCRIPTION
**Pull request by charrizard47**
Rephrased parameter section preamble
Changed to proper bullet points
Removed redundant diagrams

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -1167,39 +1141,43 @@ If you find that you are unable to execute this command successfully, there are
...
 
-`TAG` should only contain letters or numbers. +
+* `TAG` should only contain letters or numbers. +
 e.g. `monday` or `obese200kg`
 
-|`[s/SPORT]` + | `SPORT` is the sport of the clients you want to match and list. +
+|`[s/SPORT]` + a|
 
-`SPORT` is case-insensitive. +
+* SPORT` is the sport of the clients you want to match and list.
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: `SPORT`

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -378,44 +368,39 @@ This section summarises the format and important information to note while using
...
 
 * `TARGET_WEIGHT` must take the value of a whole or decimal number(eg. 65 or 86.22)
 
 * `TARGET_WEIGHT` can only have a maximum of 3 digits before the decimal place and a maximum of 2 digits after the decimal place (eg. 101.25 or 120.20)
 
-* `TARGET_WEIGHT` is optional.
-| `[h/HEIGHT]` + |
+| `[h/HEIGHT]` + a|
 
 * Substitue `HEIGHT` with the height of the client.
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: spelling mistakes with substitue

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -463,28 +444,21 @@ image::AddExampleCommand.png[]
 image::AddExampleSuccess.png[]
 
 ==== Common errors/ problems:
-If you are facing errors or difficulty adding a client, you can refer to the common errors and problems listed below and resolve your error using the solution given.
+Here are some common problems you might encounter and the solutions.
 
 . *Compulsory fields missing*
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: might want to remove the number bullet as mentioned by Ms Diana

**Review by Yonggiee — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -317,29 +317,25 @@ After setting up the program, the first thing you might want to do is to add you
 
 ==== Parameters
 
-This section summarises the format and important information to note while using the `add-c` command.
+This section shows the parameters of `add-c` and the imporant things to note about them:
```
Comment (ziyingli) [inline docs/UserGuide.adoc]: Spelling of `important`

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -348,27 +344,21 @@ This section summarises the format and important information to note while using
...
 * Substitute `ADDRESS` as the address of the client.
 
 * `ADDRESS` can take any value, but it should not be blank, or start with a whitespace.
 
-* `ADDRESS` is compulsory.
-
-| `[g/GENDER]` + |
+| `[g/GENDER]` + a|
 
 * Substitue `GENDER` with the gender of the client.
```
Comment (ziyingli) [inline docs/UserGuide.adoc]: Spelling of substitute (ctrl-f and find all haha got quite a few)

**Review by ziyingli — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -348,95 +344,80 @@ This section summarises the format and important information to note while using
...
 
 * `ADDRESS` can take any value, but it should not be blank, or start with a whitespace.
 
-* `ADDRESS` is compulsory.
+| `[g/GENDER]` + a|
 
-| `[g/GENDER]` + |
-
-* Substitue `GENDER` with the gender of the client.
+* Substitude `GENDER` with the gender of the client.
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: Spelling mistake for `substitude`

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -1167,39 +1141,43 @@ If you find that you are unable to execute this command successfully, there are
...
 
 *Format*: `filter-c [t/TAG]... [s/SPORT]...`
 
 ==== Parameters
+This section shows the parameters of `add-c` and the important things to note.
 
 [options='header']
 [cols= "15%,85%"]
 |===
 |Parameter | Important notes
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: Consider `Important points to note` for consistency

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -463,29 +444,22 @@ image::AddExampleCommand.png[]
...
+*Compulsory fields missing*
 +
 If you are adding a client and miss out any of the compulsory parameters. You will not be able to add the client.
 +
 For example, when you want to add a `Rachel Tan` into FitBiz but did not include the compulsory field `a/ADDRESS`
 +
-image::AddNoAddress.png[]
-+
-After pressing enter, the following error message will be shown.
+After entering the command, the following error message will be shown.
```
Comment (Yonggiee) [inline docs/UserGuide.adoc]: The `+` here is shown

**Review by Yonggiee — CHANGES_REQUESTED** [review]

**Review by Yonggiee — APPROVED** [review]
LGTM

**Review by ziyingli — APPROVED** [review]
LGTM

**Pull request merged**